enferno/utils/backup_utils.py

```
@@ -13,20 +12,22 @@
             "-h",
-            f"{Config.get("POSTGRES_HOST")}",
+            f"{Config.get('POSTGRES_HOST')}",
             "-U",
-            f"{Config.get("POSTGRES_USER")}" "-f",
+            f"{Config.get('POSTGRES_USER')}",
+            "-f",
             f"{filepath}",
         ]
-        return check_output(cmd, env={"PGPASSWORD": Config.get("POSTGRES_PASSWORD")})
+        env = {**os.environ, "PGPASSWORD": Config.get("POSTGRES_PASSWORD")}
+        return check_output(cmd, env=env)
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)